### PR TITLE
My first File

### PR DESCRIPTION
**Pull request by anoopvarghese-ui**
Done the master screens for customer search, add, edit, view and delete

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -26,14 +26,12 @@
             "styles": [
               "src/styles.css",
               "node_modules/datatables.net-dt/css/jquery.dataTables.css",
-              "node_modules/bootstrap/dist/css/bootstrap.min.css",
+              "node_modules/bootstrap/dist/css/bootstrap.min.css"
             ],
             "scripts": [
-            "node_modules/jquery/dist/jquery.js",
-            "node_modules/datatables.net/js/jquery.dataTables.js",
-            "node_modules/bootstrap/dist/js/bootstrap.js"
```
Comment (ramaprabhur) [inline angular.json:34]: Why bootstrap.js is removed ?

File: src/app/app.module.ts [hunk 2]
```diff
@@ -1,24 +1,21 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-import {DataTablesModule} from 'angular-datatables';
-import { FormsModule, ReactiveFormsModule } from '@angular/forms';
-import { AppRoutingModule } from './app-routing.module';
-import { MaterialModule } from './material.module';
```
Comment (ramaprabhur) [inline src/app/app.module.ts:6]: why these are removed

File: src/app/employee-list/add-search-modal/add-search-modal.component.ts [hunk 3]
```diff
@@ -35,8 +35,8 @@ public grade = new FormControl();
 public academics = new FormControl();
 
   constructor(private masterService: MasterService,
-    public dialogRef: MatDialogRef<AddSearchModalComponent>,
-    @Inject(MAT_DIALOG_DATA) public data: DialogData) { }
```
Comment (ramaprabhur) [inline src/app/employee-list/add-search-modal/add-search-modal.component.ts:39]: MAT_DIALOG_DATA is removed here, should you remove it from the import as well ?

File: angular.json (hunk 1 above)
Comment (anoopvarghese-ui) [reply angular.json:34]: Mistakenly removed, will add in the next update

File: src/app/app.module.ts (hunk 2 above)
Comment (anoopvarghese-ui) [reply src/app/app.module.ts:6]: Moved to shared module

File: src/app/employee-list/add-search-modal/add-search-modal.component.ts (hunk 3 above)
Comment (anoopvarghese-ui) [reply src/app/employee-list/add-search-modal/add-search-modal.component.ts:39]: Removed and will update in the push

**Pull request merged**